public/js/pimcore/object/classes/data/multiselect.js

```diff
@@ -99,13 +103,147 @@ pimcore.object.classes.data.multiselect = Class.create(pimcore.object.classes.da
             return stylingItems;
         }
 
-        const valueStore = new Ext.data.JsonStore({
+        const optionsStore = new Ext.data.JsonStore({
             fields: ["key", {name: "value", allowBlank: false}],
             data: datax.options
         });
+        let optionsGrid = this.createoptionsgrid(optionsStore);
+        this.optionsModel = optionsGrid.getSelectionModel();
+
+        const defaultValueStore = new Ext.data.JsonStore({
+            fields: ["value"],
+            data: datax.defaultValue
+        });
+        let defaultValueGrid = this.createdefaultvaluegrid(defaultValueStore);
+        this.defaultValueModel = defaultValueGrid.getSelectionModel();
+
+        let appendedStylingItems = stylingItems.concat([
+            {
+                xtype: "numberfield",
+                fieldLabel: t("maximum_items"),
+                name: "maxItems",
+                value: datax.maxItems,
```

Comment:
this can be bypassed by the default value section, so if the limit is 2 and the default are more than 2, eg.4, it would pre-select 4.

Reply:
resolved by https://github.com/pimcore/admin-ui-classic-bundle/pull/252/commits/1433649aeaeeaf837c8cd3975bc68d20fc1861eb